src/commands/cheats.ts

```diff
@@ -48,7 +48,7 @@ const cheatInput = (hideMessage: boolean): Promise<string> =>
         choices: choices,
         message: hideMessage ? '\n' : title,
       },
-    ]).then((answer: { code?: string }) => {
+    ]).then((answer: any) => {
```

Comment:
For some reason specifying a type on the `answer` value [doesn't work in node 7](https://travis-ci.org/wildbit/postmark-cli/builds/550413684?utm_source=github_status&utm_medium=notification).